### PR TITLE
pass on IPython Kernel example notebooks

### PR DESCRIPTION
**Pull request by minrk**
- Python 3 fixes
- add `%matplotlib notebook` to plotting example
- various copy-editing

### CODE REVIEW

**Comment by jdfreder** [comment]
I'll make sure the content in this branch runs on my end too.

File: examples/IPython Kernel/Index.ipynb [hunk 1]
```diff
@@ -91,7 +91,19 @@
...
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<a href='foo.py' target='_blank'>foo.py</a><br>"
+      ],
+      "text/plain": [
+       "/Users/minrk/dev/ip/mine/examples/IPython Kernel/foo.py"
```
Comment (jdfreder) [inline examples/IPython Kernel/Index.ipynb]: This file wasn't uploaded to the repository.

Comment (jdfreder) [inline examples/IPython Kernel/Index.ipynb]: The py files that exist in this directory&repository are 
```
example-demo.py
ipython-get-history.py
```

**Comment by jdfreder** [comment]
Other than the one comment about the missing file, everything works a-ok. :+1:

File: examples/IPython Kernel/Cell Magics.ipynb [hunk 2]
```diff
@@ -93,12 +99,17 @@
...
         "lsmagic": "BasicMagics",
+        "lx": "Other",
         "macro": "ExecutionMagics",
         "magic": "BasicMagics",
         "man": "KernelMagics",
         "matplotlib": "PylabMagics",
+        "mkdir": "Other",
         "more": "KernelMagics",
+        "mv": "Other",
+        "namespace": "Other",
```
Comment (Carreau) [inline examples/IPython Kernel/Cell Magics.ipynb]: Are some of these magics specific to you ?

File: examples/IPython Kernel/Index.ipynb (hunk 1 above)
Comment (minrk) [inline examples/IPython Kernel/Index.ipynb]: fixed (will push shortly)

File: examples/IPython Kernel/Cell Magics.ipynb (hunk 2 above)
Comment (minrk) [inline examples/IPython Kernel/Cell Magics.ipynb]: Yup, re-ran without my custom config

**Comment by minrk** [comment]
cleaned up a few artifacts, comments should be addressed.

**Pull request merged**